### ISSUE TITLE
Provide packages for ARM architectures

### ISSUE THREAD

**Issue by imrehg** [balena-io/etcher#2870]
Should provide packages which can be used e.g. on Raspbian to burn image when running on a Raspberry Pi. Currently should be possible to build from source, but packages would make things a lot easier for users.

**Comment by lurch** [comment]
Duplicate of #2748 ? (amongst other similar requests)

**Comment by jpie726** [comment]
Any progress yet?

**Comment by Botspot** [comment]
One user on the Pi forums has compiled an [Etcher armhf deb]Etcher .deb for Raspberry Pi(https://www.raspberrypi.org/forums/viewtopic.php?f=62&t=255205&start=25#p1564958).
Installation is easy:
`wget https://github.com/futurejones/balena-etcher-arm/releases/download/v1.5.63/balena-etcher-electron_1.5.63_Raspbian_Buster_armhf.deb`
`sudo apt-get install ./balena-etcher-electron_1.5.63_Raspbian_Buster_armhf.deb`

**Comment by kkarhan** [comment]
Any developments on this issue?

After all it should just be another build target in the existing pipeline...
https://github.com/balena-io/etcher/issues/2870#issuecomment-572798550

**Comment by Botspot** [comment]
[This user on github](https://github.com/Itai-Nelken/Etcher-arm-32-64) has compiled Etcher debs and has an [automated script](https://github.com/Itai-Nelken/Etcher-arm-32-64) to do so. It would be great if this was made official!

**Comment by theofficialgman** [comment]
@Botspot please delete/edit your posts here that refer to balena-etcher-electron.
it is being referenced in other posts and users who don't look hard will end up with this bad out of date package that doesn't match the upstream naming scheme

**Comment by Botspot** [comment]
Done.

**Comment by theofficialgman** [comment]
@dfunckt issue should be reopened. No armhf and arm64 Linux packages are being produced

**Comment by dfunckt** [comment]
You are right of course, in my haste I mistook the issue to be about Apple Silicon builds and didn’t read the discussion.

I spend a lot of time trying to get cross compilation going but couldn’t and I had to move on. I haven’t given up though, will give it another go as soon as I get a chance.

**Comment by Kha1r0** [comment]
Any updates on this?

**Comment by Abhimanyu14** [comment]
I am using `Etcher for macOS (arm64)` provided [here](https://etcher.balena.io/#download-etcher) on a Mac Mini with M2. 

I format the USB using GUID, but after flashing using Etcher, it changes to Master Boot Record (MBR), and the USB is not shown in the Startup screen.

**Comment by Kha1r0** [comment]
> I am using `Etcher for macOS (arm64)` provided [here](https://etcher.balena.io/#download-etcher) on a Mac Mini with M2.
> 
> I format the USB using GUID, but after flashing using Etcher, it changes to Master Boot Record (MBR), and the USB is not shown in the Startup screen.

This is due to how ISOs work. The ISO file you're using to flash your USB drive contains information about the partition table and file system to use. So your ISO probably just says to use MBR and a file system macOS can't read.

But thats completely besides the point of this thread. The macOS Arm package is here, so this is more or less only about Windows and Linux now.

**Comment by Abhimanyu14** [comment]
Hi @Kha1r0,
I am using Ubuntu ARM ISO - https://ubuntu.com/download/server/arm.

If this is not the correct place, can you please let me know what I should do or where I should check further on this?
Thank You.

**Comment by Kha1r0** [comment]
It seems your USB Drive is just ready to be used. Is it not working?

**Comment by Abhimanyu14** [comment]
I see these after formatting/erasing my USB device.
<img width="878" alt="image" src="https://github.com/user-attachments/assets/dd7e6826-dec1-44f5-a006-79a3660d735a">
<img width="874" alt="image" src="https://github.com/user-attachments/assets/e13b868d-4c4c-4de8-9f1c-20949f8d0307">

Etcher from here - "Etcher for macOS (arm64)"
https://etcher.balena.io/#download-etcher

Ubuntu from here - https://ubuntu.com/download/server/arm

**Comment by Kha1r0** [comment]
Oh. You‘re trying to install Linux on your apple silicon mac. You have to use https://asahilinux.org/ to do that. Please inform yourself thoroughly as this is not an easy process. But as i said, we‘re completely off the topic of this issue.

Sorry @imrehg for congesting your issue.

**Comment by AIWintermuteAI** [comment]
Finding this in mid-2026 from my Snapdragon Elite X Thinkpad...
With Chrome and many others waking up to consumer Linux arm64, maybe it is time to follow the lead?
